ArduPlane/tailsitter.cpp

```diff
@@ -413,7 +429,6 @@ bool Tailsitter::transition_vtol_complete(void) const
         return true;
     }
     // still waiting
-    quadplane.attitude_control->reset_rate_controller_I_terms();
```

Comment:
this change is not related to this PR.

Reply:
oops...will fix..branched off wrong place apparently